prow/images/cleaner/README.md

```diff
@@ -0,0 +1,15 @@
+# Cleaner Docker Image
+
+## Overview
+This image contains script which perform cleanup of service account profile in the `kyma-project` project. 
+Script requires environment variable `GOOGLE_APPLICATION_CREDENTIALS` which is a path to service account key.
+
+## Build
```

Comment:
```suggestion
## Installation
```

Reply:
I know that in some other README.mds it is "Build" but we try to follow the general template and it states otherwise - "Build "headings will be replaced.